src/logging.cc

```diff
@@ -1459,7 +1459,7 @@ static void logging_fail() {
 #if defined(_DEBUG) && defined(_MSC_VER)
   // When debugging on windows, avoid the obnoxious dialog and make
   // it possible to continue past a LOG(FATAL) in the debugger
-  _asm int 3
+  __debugbreak();
```

Comment:
It compiles without it. 
As far as I know, <windows.h> includes it, so maybe windows/port.h is included somehow.

To be sure, the function "DebugBreak()" from windows.h can be used:
https://msdn.microsoft.com/library/windows/desktop/ms679297.aspx

However, the official replacement (according to https://msdn.microsoft.com/en-us/library/45yd4tzz.aspx) for _asm int 3 is __debugbreak().